src/include/allsky_common.h

```diff
@@ -0,0 +1,95 @@
+// Items used by the "capture" and other programs.
+
+// Image formats.  Match the ASI_* settings to make it consistent with ZWO's library.

```

Comment:
ASI stands for what ? (ZWO  camera model or Allsky Interface)
in the future we should sperate some defines or variables for the different cameras...

Reply:
sorry, I should have replied here and not via personal email.